docs/getting-started/my-account/README.md

```diff
@@ -122,32 +119,84 @@ Click the grey **Register device** button to register an authentication app. Ope
 
 Click the blue **Continue** button to finish the registration.
 
-![register authentication app](1572879640187.png)
+![openproject_my_account_authenticator_app](openproject_my_account_authenticator_app.png)
+
+## Access tokens
+To view and manage your OpenProject access tokens navigate to **My account** and choose **Access tokens** from the menu. 
+Access tokens allow you to grant external applications access to resources in OpenProject. 
+
+![openproject_my_account_access_tokens](openproject_my_account_access_tokens.png)
 
+### API
+API tokens allow third-party applications to communicate with this OpenProject instance via REST APIs. If you have not yet created an API token, this list will be empty. You may need admin privileges to be able to create an API.
 
+You can enable an API token under *Administration -> API and webhooks*. LINK TO HOW ITS DONE IN SYSTEM ADMIN GUIDE.
+### iCalendar
+iCalendar tokens allow users to subscribe to OpenProject calendars and view up-to-date work package information from external clients. 
+This list will be empty if you have not yet subscribed to any calendars. Once you [subscribe to a calendar] (LINK TO ICAL DOCUMENTATION), a list of all the calendars that you have subscribed to will appear here. The name of the calendar is clickable and will lead you directly to the respective calendar in OpenProject. 
+
+![OpenProject calendar list under my account](openproject_my_account_access_tokens_calendar_list.png)
+
+You can delete an entry in the iCalendar list by clicking on the **Delete** icon. This will trigger a warning message asking you to confirm the decision to delete.  
```

Comment:
I maybe would add that by deleting this token the user will no longer have access to the OpenProject information in all the linked clients using this token.